test/e2e/run_test.go

```diff
@@ -1406,6 +1406,13 @@ VOLUME %s`, ALPINE, volPath, volPath)
 		Expect(session.OutputToString()).To(ContainSubstring("1001-123"))
 	})
 
+	It("podman run --mount type=devpts,target=/dev/pts with ptmxmode", func() {
+		session := podmanTest.Podman([]string{"run", "-t", "--mount", "type=devpts,target=/dev/pts,ptmxmode=0666", fedoraMinimal, "findmnt", "-noOPTIONS", "/dev/pts"})
```

Comment:
`0666` is the default mode so we cannot be sure that we actually passed the cli value to the runtime.
Maybe pass something non default such as `ptmxmode=0400`

Also the `-t` is not needed and should be dropped as it just causes terminal control chars to be added to the output.

Reply:
I made the suggested changes and pushed a new version.